packages/host/app/lib/matrix-classes/room.ts

```diff
@@ -44,6 +44,14 @@ export default class Room {
       ?.name;
   }
 
+  get memberIds(): string[] {
```

Comment:
this seems like something that should be using `@cached` no? every call to this property will run this logic